code/game/gamemodes/blob/overmind.dm

```diff
@@ -15,18 +15,30 @@
 	var/blob_points = 0
 	var/max_blob_points = 100
 	var/last_attack = 0
+	var/datum/reagent/blob_reagent = "water"
+	var/datum/reagent/blob_reagent_name = "Water"
+	var/datum/reagent/blob_color = ""

```

Comment:
you only need the blob_reagent_datum var, the rest can be obtained using that without separate vars.